extensions/wikia/DesignSystem/i18n/build.js

```diff
@@ -7,8 +7,7 @@ var fs = require('fs'),
 	messageParamsMapping = {
 		'global-footer-licensing-and-vertical-description': {
```

Comment:
Can we remove "licensing" from this key name since it's been moved out?

Reply:
there is already ticket to do that in our backlog opened by platform. I don't want to introduce such change in this PR since it requires updating all of DS clients